04/05/2023 03:36:21 PM [INFO] Added parameter: Conv2DSize: min=1, max=256, is_integer=True, resolution=1, stages=6
04/05/2023 03:36:21 PM [INFO] Added parameter: Dense1Size: min=512, max=1024, is_integer=True, resolution=1, stages=8
04/05/2023 03:36:21 PM [INFO] Added parameter: Dropout1: min=0.1, max=0.9, is_integer=False, resolution=0.1, stages=2
04/05/2023 03:36:21 PM [INFO] Beginning run:
<ltpspace.Explorer object at 0x000001D9A1D03D10>
04/05/2023 03:36:21 PM [INFO] Recovering from: Logfiles\ltps_2023-04-05_15-33-34.log
04/05/2023 03:36:21 PM [INFO] Recovered score: 0.7430728646318615
04/05/2023 03:36:21 PM [INFO] Recovered params: [213, 820, 0.8]
04/05/2023 03:36:21 PM [INFO] Recovered currentparam: 1
04/05/2023 03:36:21 PM [INFO] Recovered direction: True
04/05/2023 03:36:21 PM [INFO] Recovered stages: 7
04/05/2023 03:36:21 PM [INFO] Beginning Parameter Space Search...
04/05/2023 03:36:21 PM [INFO] Status Update - Score: 0.7430728646318615; Parameters: [213, 820, 0.8]; Currentparam: 1; Direction: True; Stage: 7;
04/05/2023 03:36:21 PM [DEBUG] Trying Dense1Size from 820 to 948
04/05/2023 03:36:24 PM [ERROR] Unexpected error occurred: in user code:

    File "C:\Users\cpnbe\AppData\Roaming\Python\Python311\site-packages\keras\engine\training.py", line 1284, in train_function  *
        return step_function(self, iterator)

    TypeError: outer_factory.<locals>.inner_factory.<locals>.tf____call__() missing 1 required positional argument: 'y_true'

